MinecraftClient/Inventory/ItemMovingHelper.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace MinecraftClient.Inventory
+{
+    /// <summary>
+    /// Class that contains useful methods to move item around in a container
+    /// </summary>
+    public class ItemMovingHelper
+    {
+        private Container c;
+        private Func<int, int, WindowActionType, bool> windowAction;
```

Comment:
Any reason for storing a callback to windowAction here and not just a reference to McClient itself? 🤔

Reply:
Yes. Since ChatBot cannot access to McClient

Reply:
Oh, OK. In that case, how adding this to the ChatBot API just after the `WindowAction` API method?
```C#
        /// <summary>
        /// Get inventory action helper
        /// </summary>
        /// <param name="container">Inventory Container</param>
        /// <returns>ItemMovingHelper instance</returns>
        protected ItemMovingHelper GetItemMovingHelper(Container container)
        {
            return new ItemMovingHelper(Handler, container);
        }
```
Would make the ItemMovingHelper more visible and easy to use for people using the ChatBot API.

Reply:
Okay.